src/backend/distributed/master/master_create_shards.c

```diff
@@ -241,7 +271,7 @@ CreateShardsWithRoundRobinPolicy(Oid distributedTableId, int32 shardCount,
  * the source relation.
  */
 void
-CreateColocatedShards(Oid targetRelationId, Oid sourceRelationId)
+CreateColocatedShards(Oid targetRelationId, Oid sourceRelationId, bool emptyTable)
```

Comment:
Same comment as above: I think the name `emptyTable` holds no particular relevance to this function. Maybe `useExclusiveConnections`.